slack_notification/slack_notification.php

```diff
@@ -143,10 +171,18 @@ function _get_secrets($requiredKeys, $defaults)
 function _slack_notification($slack_url, $channel, $username, $text, $attachment, $alwaysShowText = false)
 {
   $attachment['fallback'] = $text;
-  $post = array(
-    'username' => $username,
+  if (substr($channel,1) == '#') {
+    // Post with the $username
+    $post = array(
+      'username' => $username
+    );
+  } else {
+    // Do not post with the username for private channels
+    $post = array();
+  }
+  $post += array(
     'channel' => $channel,
-    'icon_emoji' => ':lightning_cloud:',
+    'icon_emoji' => ':pantheon:',
```

Comment:
Good point! We should probably convert this to a full url to a 128x128 Pantheon Emoji. Slack accepts full image urls for this field.